index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header id="header">
+            <nav class="navbar">
+                <h2>My Logo</h2>
+                <img src="images/Hamburger_button.svg" alt="menu button">
+            </nav>
+    </header>
+    
+    <section id="headline">
+        <h2>I’m Mariane Glad to see you!</h2>
```

Comment:
- [ ] Kindly, replace `<h2>` with `<h1>` it is semantically correct for titles 💯 , and also if you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. 🥇